docs/schema.md

```diff
@@ -259,7 +259,7 @@ In `v1`, images are declared inside the `properties` section, as follows:
 
 ```yaml
 properties:
-  mainImageName:  # A primary image is required.
```

Comment:
Can you also update L170?

Reply:
Saw this right after clicking "Merge" - sending a follow-up PR.